tigergraph/k8s/README.md

```diff
@@ -35,27 +35,68 @@ kubectl apply -f ./deploy/tigergraph-gke.yaml
+
+Following the instructions on AWS documentation to add EBS CSI add-on before proceed.
+https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html
+
+Use ```kubectl get pods -n kube-system``` to check if EBS CSI driver is running. An example output is
+```
+NAME                                  READY   STATUS    RESTARTS        AGE
+...
+coredns-5948f55769-kcnvx              1/1     Running   0               3d6h
+coredns-5948f55769-z7mbr              1/1     Running   0               3d6h
+ebs-csi-controller-75598cd6f4-48dp8   6/6     Running   0               3d4h
+ebs-csi-controller-75598cd6f4-sqbhw   6/6     Running   4 (2d11h ago)   3d4h
+ebs-csi-node-9cmbj                    3/3     Running   0               3d4h
+ebs-csi-node-g65ns                    3/3     Running   0               3d4h
+ebs-csi-node-qzflk                    3/3     Running   0               3d4h
+ebs-csi-node-x2t22                    3/3     Running   0               3d4h
+...
+```
+
+If the ebs csi driver is installed, then run (here use "tigergraph" as namespace for an example)
 ```bash
-./tg eks kustomize -s 2 --pv 280 --cpu 30 --mem 200 -l [license string]
-kubectl apply -f ./deploy/tigergraph-eks.yaml
+kubectl create ns tigergraph
+git clone https://github.com/tigergraph/ecosys.git
+cd ecosys/k8s
+./tg gke kustomize -v 3.7.0 -n tigergraph -s 4 --pv 700 --cpu 30 --mem 200 -l [license string]
+kubectl apply -f ./deploy/tigergraph-eks-tigergraph.yaml
 ```
 
+
 ## Verify deployment
 
-Deployment can take several minutes. Use `kubectl get pod` to verify the deployment. An example output is
+Deployment can take several minutes. Use `kubectl get pod -n tigergraph` to verify the deployment. An example output is
 
 ```
 NAME              READY   STATUS    RESTARTS   AGE
 installer-cztjf   1/1     Running   0          5m23s
 tigergraph-0      1/1     Running   0          5m24s
 tigergraph-1      1/1     Running   0          3m11s
 ``` 
-## Download data
+Alternative verification commands include:
+```
+kubectl get all -n tigergraph
+kubectl describe pod/tigergraph-0 -n tigergraph
+kubectl describe pvc -n tigergraph
+
+```
 
+
+## Download data
+To download the data, service key json file must be located in ```k8s/``` . The bucket is public now and any service key should work.
 1. Fill in the parameters in `vars.sh`.
     * `NUM_NODES` - number of nodes.
     * `SF` - data source, choices are 100, 300, 1000, 3000, 10000.
-    * `DOWNLOAD_THREAD` - number of download threads
+    * `DOWNLOAD_THREAD` - number of download threads    
+         
+
+
+         
+1. Make sure to put your own service key file in ```tigergraph/``` folder
+
+    Service key file with view only permission can be provided to auditor upon request.
```

Comment:
This repo is for public. You can say, our bucket is public, and any google cloud service key is able to access the data. To create service key, refer to [Google Cloud documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).